packages/conf-libclang/conf-libclang.19/opam

```diff
@@ -0,0 +1,59 @@
+opam-version: "2.0"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "The LLVM team"
+homepage: "http://llvm.org"
+bug-reports: "https://llvm.org/bugs/"
+license: "MIT"
+build-env: [
+  [HOMEBREW_NO_AUTO_UPDATE = "1"]
+]
+build: [
+  ["bash" "-ex" "configure.sh" version] # pass pkg var '19' to test <= 19.0.x
+]
+depends: [
+  "conf-bash" {build}
+]
+depexts: [
+  ["llvm@19"] {os-distribution = "homebrew" & os = "macos"}
+  ["llvm-19"] {os-distribution = "macports" & os = "macos"}
+  ["llvm" "clang"] {os-distribution = "arch"} #currently llvm is 21.1.6-1 https://archlinux.org/packages/extra/x86_64/llvm/
```

Comment:
```suggestion
  ["llvm19" "clang19"] {os-distribution = "arch"}
```
CI is currently failing on Arch as it is installing the default version 21.
We will need to name version 19:
https://archlinux.org/packages/extra/x86_64/llvm19/
https://archlinux.org/packages/extra/x86_64/clang19/

IIRC, Arch is a rolling distribution, meaning that at some future point these are going to fail because they are removed from the distribution :shrug: